src/test/java/org/json/junit/JSONParserConfigurationTest.java

```diff
@@ -43,4 +103,14 @@ public void verifyMaxDepthThenDuplicateKey() {
         assertTrue(jsonParserConfiguration.isOverwriteDuplicateKey());
         assertEquals(42, jsonParserConfiguration.getMaxNestingDepth());
     }
+
+    private List<String> getNonCompliantJSONList() {
+        return Arrays.asList(
+            "[1,2];[3,4]",
```

Comment:
Please add to this list an array with string elements, including an unquoted string.
Also please add a JavaDoc comment for this method explaining what it is used for.

Reply:
Done.

Answering your question, I added a new JSON file test resource which contains a compliant complex structure with various levels which is used in givenCompliantJSONArrayFile_testStrictModeTrue_shouldNotThrowAnyException

I'll be happy to add more test cases if you want.